plugins/gcloud/app/controllers/gke_clusters_controller.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+class GkeClustersController < ApplicationController
+  before_action :authorize_super_admin!
+
+  def new
+  end
+
+  def create
+    project = params.require(:gke_cluster).require(:gcp_project)
+    cluster = params.require(:gke_cluster).require(:cluster_name)
+    zone = params.require(:gke_cluster).require(:zone)
+
+    # prepare the file gcloud will write into
+    folder = ENV.fetch("GCLOUD_GKE_CLUSTERS_FOLDER")
```

Comment:
Might be nice to show an error on the `new` page if this ENV var isn't set, rather than firing off an Airbrake error when you try to create.

Reply:
Ah, nevermind. I see that you do that check below in the view that makes the button appear.